bounding_box:

  bounding_box_height: 260
  bounding_box_width: 296
  bounding_box_center_x: 198
  bounding_box_center_y: 130

gaze_lock_commander:

  distance_to_object: 0.269
  distance_to_object_step: 0.
  sharpness_fun_type: "image_area"

heuristic_commander:

  # ground_truth_distance: 0.489 # On table
  # ground_truth_distance: 0.425 # On cube
  ground_truth_distance: 0.2695 # On cylinder

  current_heuristic_mode: cycle_sinX # cycle_circularCW // cycle_sinY // cycle_circularCW // cycle_circularCCW // cycle_smoothConstantX
  # current_heuristic_mode: cycle_sinY+perturbation_sinZ

  heuristic_cycle:
    gain_heuristic_cycle_control_commands: 0.1 # 0.1
    freq_heuristic_cycle_control_commands: 0.5

  heuristic_perturbation:
    gain_heuristic_perturbation_control_commands: 0.0

